### DISCUSSION TITLE
【灌水】正在研究绳网发送链接和图片的规则

### DISCUSSION THREAD

**Discussion by C-Studio-Github** [share121/inter-knot#2649] [其他]
[我看看能发点啥](https://www.bilibili.com/video/BV1GJ411x7h7)
![屏幕截图 2024-08-18 114229](https://github.com/user-attachments/assets/6d4b02d2-fc3e-486b-81b3-8c7bb33d05b9)

**Comment by wwwsheepwww** [comment]
我只知道我被骗了（悲

**Comment by linexzhou** [comment]
我就知道<img src="https://bbs-static.miyoushe.com/static/2024/07/31/73346f2b98d45b62da9139d2774fa856_8641114165088315754.png" alt="三月七 告退" width="75" >
